docs/presentations/2024-08-27-CU-Anschutz-DBMI-Retreat/poster.qmd

```diff
@@ -0,0 +1,154 @@
+---
+title: Single-cell Morphology Quality Control (coSMicQC)
+format:
+  poster-typst:
+    size: "48x36"
+    poster-authors: "Dave Bunten¹, Jenna Tomkinson¹, Gregory Way¹"
```

Comment:
In your opinion, would we be considered co-first authors for this or would this only apply to a future manuscript? Just curious! I really like how easy and seamless this is!

Reply:
Thanks, good question - I don't know enough about how authorship functions in the scope of posters but I imagine it'd make sense for us to be co-first authors. Should I reformat this to be reordered or use an ampersand?

Reply:
Keep yourself first between the two of us since you will be the main presenter of the work (either poster or podium), so using an ampersand would be perfect!

Reply:
Sounds good, I made an adjustment here to use an asterisk instead with a symbol key addition below the institution in the title (this seemed to be more formal and might align with any future manuscript work we do).